recipes/wx/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "wxPython" %}
+{% set version = "4.0.0a2" %}
+{% set sha256 = "3acd5160d69f56b588476d1907eff67b53ad58fb2eba4dedabcdc3490d170338" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [unix]
+
+requirements:
+  build:
+    - python
+    - six
+    - setuptools
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - wx
+
+about:
+  home: https://wxpython.org/Phoenix/
+  license: wxWidgets Library
+  license_family: LGPL
```

Comment:
It's weird the don't add it here: https://github.com/wxWidgets/Phoenix 😕

Could you please raise an issue asking them to include the license file in the Python source code and to please make sure the license file is added to the `MANIFEST.in` so it shows up on PyPI?